nf1_cellprofiler_classifier_fs_all_cellprofiler_all_feat_perfect.ipynb

```diff
@@ -0,0 +1,499 @@
+{
```

Comment:
<p>Isn't a value of 1.0 not a good sign? I remember speaking about this with you briefly so if you could explain why 1.0 would be good to have for this measurement, that would help me a lot for understanding!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/NF1_SchwannCell_data_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
@MattsonCam - Adding to what @jenna-tomkinson mentions, I thought 100% accuracy looked concerning due to possible [overfitting](https://en.wikipedia.org/wiki/Overfitting). If it's expected, consider providing some documentation within the notebook describing why and code to explore overfitting aspects.

Reply:
It could be due to overfitting indirectly, in part, because the test accuracy is perfect. There might be a sampling issue, but the way I am sampling and training should be okay. The sample size might be causing this. I'll probably modify my approach for future models.

I commented below about adding this to the documentation, just to let you know.